signer/migrations/0003__create_tables.sql

```diff
@@ -26,8 +26,6 @@ CREATE TABLE sbtc_signer.stacks_blocks (
     bitcoin_anchor BYTEA NOT NULL,
     created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
 );
--- Index to serve queries filtering on `parent_hash`. This is commonly used when
```

Comment:
This index is added in a later script, but the comment got left here apparently, just cleaning up.